astroquery/first/tests/test_first.py

```diff
@@ -0,0 +1,63 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
+
+import os
+
+import numpy.testing as npt
+import pytest
+import astropy.units as u
+
+from ...utils import commons
+from ...utils.testing_tools import MockResponse
+from ... import first
+
+DATA_FILES = {'image': 'image.fits'}
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), 'data')
+    return os.path.join(data_dir, filename)
+
+
+@pytest.fixture
+def patch_parse_coordinates(request):
+    def parse_coordinates_mock_return(c):
+        return c
+    try:
+        mp = request.getfixturevalue("monkeypatch")
+    except AttributeError:  # pytest < 3
```

Comment:
we don't support python2.7 any more, so no need for the try/except

Reply:
Thanks, I made these changes (I inherited the Python2 compatibility code from the magpis module).